pgt/src/PGT/Output/Text.hs

```diff
@@ -0,0 +1,35 @@
+module PGT.Output.Text
+  ( parseName
+  )
+where
+
+import Data.Attoparsec.Text (Parser)
+import PGT.Prelude
+
+import qualified Data.Attoparsec.Text as Text
+import qualified Data.Char            as Char
+import qualified Data.Text            as Text
+
+parseName :: Parser Text
+parseName = do
+  head <- Text.takeWhile1 Char.isAlphaNum
+  tail <- Text.many' parseTail
+
+  pure . Text.concat $ head : tail
+  where
+    parseTail :: Parser Text
+    parseTail = do
+      char      <- parseSeparator
+      remaining <- Text.takeWhile1 Char.isAlphaNum
```

Comment:
__suggestion: (nitpick)__ The expression `Text.takeWhile1 Char.isAlphaNum` appears here and on line 15. Should this be a private function and reused in both places?